### PR TITLE
Media group roles

### PR DESCRIPTION
**Pull request by Thiblef**
[OO-FEATURE] Media roles gestion for each folders are available in the group panel
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/160
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/139

### CODE REVIEW

**Comment by itkg-alavieille** [comment]
:+1:  think to add comment in your PR for the changelog

**Comment by itkg-ngilain** [comment]
:+1:

**Comment by Thiblef** [comment]
@itkg-alavieille @itkg-ngilain @nicolasThal PR updated

File: ApiBundle/Transformer/GroupTransformer.php [hunk 1]
```diff
@@ -124,6 +138,19 @@ public function reverseTransform(FacadeInterface $facade, $group = null)
...
+        $mediaFolderTransformer = $this->getTransformer('media_folder_group_role');
+        if (!$mediaFolderTransformer instanceof TransformerWithGroupInterface) {
+            throw new UnexpectedValueException("Media Folder Group Role Transformer must be an instance of TransformerWithContextInterface");
+        }
+        foreach ($facade->getMediaFolderRoles() as $mediaFolderRoleFacade) {
+            $group->addMediaFolderRole($mediaFolderTransformer->reverseTransformWithGroup(
+                $group,
+                $mediaFolderRoleFacade,
+                $group->getMediaFolderRoleByMediaFolderAndRole($mediaFolderRoleFacade->folder, $mediaFolderRoleFacade->name)
+                )
```
Comment (itkg-ngilain) [inline ApiBundle/Transformer/GroupTransformer.php]: ident error here

**Comment by itkg-ngilain** [comment]
:+1: with the ident fix

**Pull request closed without merging**